books/spiders/book.py

```diff
@@ -0,0 +1,51 @@
+import scrapy
+from scrapy import Request
+from scrapy.http import Response
+
+
+class BookSpider(scrapy.Spider):
+    name = "book"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def parse(self, response: Response, **kwargs):
+        for book in response.css(".product_pod"):
+            detail_url = response.urljoin(book.css("a::attr(href)").get())
+            yield Request(
+                url=detail_url,
+                callback=self.parse_detailed_info,
```

Comment:
The callback function is incorrectly named `parse_detailed_info`. It should be `parse_book_details` to match the defined method name. Update the callback to `callback=self.parse_book_details`.